app/components/sidebars/main/channels_list/channel_item/index.js

```diff
@@ -39,6 +39,7 @@ function makeMapStateToProps() {
             } else {
                 const teammateId = getUserIdFromChannelName(currentUserId, channel.name);
                 const teammate = getUser(state, teammateId);
+                channel.delete_at = teammate.delete_at;
```

Comment:
This mutates the stored state. We'll either have to pass it in separately or pass in the teammate object itself. Or perhaps we could filter out the deleted user's DM before this point?

Reply:
true.. I'll pass it a a separate value...